tsconfig.json

```diff
@@ -18,7 +18,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "noImplicitAny": false
```

Comment:
We should not relax these constraints as leaving them on leads to better code. If you really think this flag needs to be turned off, please provide a persuasive argument for why it needs to be done.